edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -842,7 +843,8 @@ impl ModuleRuntime for DockerModuleRuntime {
             .as_ref()
             .lock()
             .expect("Could not acquire system resources lock");
-        system_resources.refresh_all();
+        system_resources.refresh_system();
```

Comment:
Since you've split this into individual `refresh_*`, can you move the `refresh_system` call right above the `get_global_processor_info` call, and the `refresh_disks` call right above the `get_disks` call, so that they're correlated just like `refresh_process` with `get_process` ?